src/github.com/getlantern/detour/whitelist.go

```diff
@@ -19,7 +19,9 @@ var (
 func AddToWl(addr string, permanent bool) {
 	muWhitelist.Lock()
 	defer muWhitelist.Unlock()
-	whitelist[addr] = wlEntry{permanent}
+	if add != "" {
```

Comment:
A bit unclear to me. `getParentDomain()` is used by `whitelisted()` but not here. `addr` will  never be "" except passed in intentionally.